ros2/nightly/platform.yaml

```diff
@@ -10,4 +10,4 @@ platform:
     maintainer_name:
     arch: amd64
     type: distribution
-    version:
```

Comment:
I think the `version` parameter was previously used for setting the package version in the apt install. Leaving it empty was to signify to the template function to poll the apt repo for the latest version number. Did you refactor this logic in docker_templates to match your repurpose of it here? Given osrf apt repos don't archive past version for long, this repurpose of `version` you've made seems more useful.

Reply:
Oh I was actually wondering why some files had it https://github.com/osrf/docker_images/blob/8404682c98c1182203bcbe9898af6832f387aaab/ros2/source/platform.yaml#L12 and some didn't https://github.com/osrf/docker_images/blob/8404682c98c1182203bcbe9898af6832f387aaab/ros2/nightly/platform.yaml#L13

I didn't change any corresponding logic in the templates scripts. Surprising the generated files didnt change, maybe it's used only for gazebo? Do you remember where this is used exactly?

Reply:
I think it was this:

https://github.com/osrf/docker_templates/blob/366392402917ff19b14fc0c5ed088bed8bd2e329/docker_templates/packages.py#L80

Reply:
Discussed this offline with @ruffsl, we will not use `version` but `ros_version` instead, that allows to not change the semantic and be consistent with the name of the environment variable `ROS_VERSION`